docs/concepts/security.md

```diff
@@ -0,0 +1,60 @@
+# Security
+
+This document explains the security aspects of Kong Ingress Controller.
+
+Kong Ingress Controller communicates with Kubernetes API-server and Kong's
+Admin API. APIs on both sides offer authentication/authorization features
+and the controller integrates with them gracefully.
+
+## Kubernetes RBAC
+
+Kong Ingress Controller is deployed with RBAC permissions as explained in
+[deployment](deployment.md) document.
+It has read and list permissions on most resources but requires update
+and create permission for a few resources to provide seamless integration.
+The permissions can be locked down further if needed depending on the specific
+use-case.
+This RBAC policy is associated with a ServiceAccount in Kubernetes, which
+takes of the Controller authenticating and authorizing against Kuberentes
+API-server.
+
+## Kong Admin API protection
+
+Kong's Admin API is used to control configuration of Kong and proxying behavior.
+If an attacker gets access to Kong's Admin API, Kong will be compromised
+and all bets are off at that point. Hence, it is important that the deployment
```

Comment:
Instead of "all bets are off at that point," which does not sound sober/objective, describe what an attacker will be able to do. 

"An attacker will be able to perform the same actions as an authorized user--including actions that they may compromise the confidentiality, integrity, or availability of data proxied through Kong."

Something along those lines, more concise if possible.